README.md

```diff
@@ -1685,6 +1687,29 @@ you are using apply and auto correct.
 
 - [CMClientSettingsComputerAgent](Source\Examples\Resources\CMClientSettingsComputerAgent\CMClientSettingsComputerAgent.ps1)
 
+### CMClientSettingsComputerRestart
```

Comment:
Please add the new resource to supported resources under the reverse dsc section

Reply:
Done